app/src/main/java/com/movies/presentation/home/ui/HomeFragment.kt

```diff
@@ -43,6 +36,21 @@ class HomeFragment : BaseFragment<HomeUIState, HomeViewModel>() {
         }
     }
 
+    private lateinit var moviePagingAdapter: MoviePagingAdapter
+
+    override fun onBind() {
+        handleResult(DETAILS) {
+            viewModel.onCreate()
```

Comment:
you have to rename onCreate

Reply:
the name is not a problem. Space also has viewmodel onCreate. The issue was that I call it from onBind as well as onCreate. It should only be called once from fragment onCreate.